plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/browser/OpenHandler.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Groovy Editor (GE) is a library that brings a groovy console to the Eclipse RCP. 
+ * GE is developed by CNRS http://www.cnrs.fr/.
+ *
+ * GE is part of the OrbisGIS project. GE is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU Lesser 
+ * General Public License as published by the Free Software Foundation;
+ * version 3.0 of the License.
+ *
+ * GE is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+ * without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details http://www.gnu.org/licenses.
+ *
+ *
+ *For more information, please consult: http://www.orbisgis.org
+ *or contact directly: info_at_orbisgis.org
+ */
+package org.orbisgis.ui.browser;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.resources.IFile;
+import org.eclipse.equinox.internal.app.Activator;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.SWTError;
+import org.eclipse.swt.browser.Browser;
+import org.eclipse.swt.layout.FillLayout;
+import org.eclipse.swt.program.Program;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Shell;
+import org.eclipse.ui.IWorkbenchWindow;
+import org.eclipse.ui.PartInitException;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.ui.browser.IWebBrowser;
+import org.eclipse.ui.browser.IWorkbenchBrowserSupport;
+import org.eclipse.ui.handlers.HandlerUtil;
+import org.eclipse.ui.part.FileEditorInput;
+import org.orbisgis.core.CoreActivator;
+import org.orbisgis.core.logger.Logger;
+import org.orbisgis.core.workspace.ICoreWorkspace;
+import org.orbisgis.ui.editors.groovy.GroovyEditor;
+import org.orbisgis.ui.editors.groovy.GroovyUtils;
+
+public class OpenHandler extends AbstractHandler {
```

Comment:
Add author
Adrien Bessy, LAB-STICC, CNRS

Reply:
This class is never used ?

Reply:
This class allows to open a web page in the software, but we don't use it.